classes/controllers/FrmSettingsController.php

```
@@ -296,6 +302,17 @@ public static function captcha_settings() {
 		include FrmAppHelper::plugin_path() . '/classes/views/frm-settings/captcha/captcha.php';
 	}
 
+	/**
+	 * Shows email settings.
+	 *
+	 * @since x.x
+	 */
+	public static function email_settings() {
+		$frm_settings = FrmAppHelper::get_settings();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$frm_settings'. (undefined)

(UnusedLocalVariable)